src/rsa_api/rsa_api.py

```diff
@@ -620,10 +623,7 @@ def CONFIG_SetFreqRefUserSetting(self, i_usstr: Union[str, None] = None) -> None
             self.err_check(self.rsa.CONFIG_SetFreqRefUserSetting(None))
         else:
             RSA.check_string(i_usstr)
-            if (
-                i_usstr == "Invalid User Setting"
-                or len(i_usstr) != _FREQ_REF_USER_SETTING_STRLEN
-            ):
+            if i_usstr == "Invalid User Setting":
```

Comment:
Would there still be a need to check that user string has length less than _FREQ_REF_USER_SETTING_STRLEN?

Reply:
Good point. Added this check in [ef39a16](https://github.com/NTIA/tekrsa-api-wrap/pull/16/commits/ef39a1602844af50c8755d6193e2e881cb41f20b), with its own informative error message